hash_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"unsafe"
 
-	verushash "github.com/Asherda/Go-VerusHash"
+	verushash "github.com/DavidLDawes/Go-VerusHash"
```

Comment:
As noted in the go.mod comment above, this will switch to Asherda/Go-Verushash and then VerusCoin/Go-Verushash as we get Go-Verushash settled.

We'll want to propogate the changes back, so I'll merge the change from DavidLDawes to Asherda back to my DavidLDawes archive once it is in, and merge the change to VereusCoin too when that hits.

Asherda/lightwalletd will need to merge the VerusCoin change as well, and Mike should merge it to his fork if he wants to use it as well. At that point all of our forks point to the correct VerusCoin/Go-VerusHash source and we can forget about it. Until we change it and need to retest, anyway.